tests/prescribed_temperature_in_field.prm

```diff
@@ -0,0 +1,64 @@
+## First we set up a unit length box model with uniform everything.
+set Dimension   = 2
+set End time    = 10000
```

Comment:
Reduce the run time for the test case?

Reply:
It still happens in 1 time step and I figured it might be more convincing that conduction does not happen if the time step is larger

Reply:
I think that makes sense.